ansible_collections/arista/avd/docs/release-notes/2.x.x.md

```diff
@@ -239,9 +239,9 @@ md-toc==7.1.0
 
 ### Data model modifications
 
-This section provides an overview of only the data model that have ***changed*** from the previous release that would require user modifications. See the release notes and role documentation for all new additions.
+This section provides an overview of only the data model that have *__changed__* from the previous release that would require user modifications. See the release notes and role documentation for all new additions.
 
-**eos_cli_config_gen:**
+__eos_cli_config_gen:__
```

Comment:
+1

Reply:
@ClausHolbechArista Do you mean whether to use asteriks or underscores?

Reply:
Currently the rule is set to consistent. If a document starts with using `__` then it will expect the author to continue that. Same if the author uses `**`, we can force one or the other. If need be I can update the rule and re run the auto fix. See [here](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md049---emphasis-style-should-be-consistent)

Reply:
Updated this to enforce asterisks!